confmap/confmaptest/configtest.go

```diff
@@ -28,7 +28,7 @@ import (
 // LoadConf loads a confmap.Conf from file, and does NOT validate the configuration.
 func LoadConf(fileName string) (*confmap.Conf, error) {
 	// Clean the path before using it.
-	content, err := ioutil.ReadFile(filepath.Clean(fileName))
+	content, err := os.ReadFile(filepath.Clean(fileName))
```

Comment:
I had to do only a small change here.

Reply:
i'm surprised this was the only use of ioutil that needed to be changed.